serverless/containers/how-to/add-a-custom-domain-to-a-container.mdx

```diff
@@ -36,11 +36,65 @@ Each container comes with an automatically generated domain, on which it is avai
+| Route `/.well-known/acme-challenge` is not handled correctly. | To perform the HTTP-01 challenge, a call we be made on `<your_custom_domain>/.well-known/acme-challenge/<token>`. If you are using a proxying/caching tool (like Cloudflare) between the custom domain and the Serverless Container endpoint, a cached version of that route can be served, making the check fails. | Don't configure caching rules for routes starting with `/.well-known/acme-challenge`. <Lightbox src="cloudflare-bypass-cache-well-known-acme-challenge.webp" size="medium" alt="Bypass cache for /.well-known/acme-challenge routes" /> |
+
+## Migrate from a domain already serving another endpoint
+
+Read this section if you are trying to migrate a domain that is already serving a live website/API/web application -- but not yet hosted on a Serverless Container -- and you are trying to migrate.
+
+To clarify, let's take a concrete example:
+
+- your website is hosted on a Scaleway instance, with the IP `51.15.x.x`
+- you own a domain `mydomain.com`
+- an A record is configured on `website.mydomain.com` and points to `51.15.x.x`
+- when a client accesses `http://website.mydomain.com`, the request hits your instance IP `51.15.x.x:80`
+- you also have a running version of your website hosted as a Serverless Container (only accessible using `prodtd9zl1mz-website.functions.fnc.fr-par.scw.cloud`), and now you want your users to access this version from `http://website.mydomain.com`
+
+Before adding the custom domain on your Serverless Container, you must change the DNS record to point to the Serverless Container endpoint:
+
+- before: `website.mydomain.com` is an A record to `51.15.x.x`
+- after: `website.mydomain.com` is a CNAME record to `prodtd9zl1mz-website.functions.fnc.fr-par.scw.cloud`
+
+By doing this, clients that already have the `website.mydomain.com` DNS record cached locally will continue to hit `51.15.x.x`, until the TTL expires. New clients (or those whose cache has expired) will start to hit `prodtd9zl1mz-website.functions.fnc.fr-par.scw.cloud`. **However**, until the custom domain is not configured on the Serverless Container, these requests will end up in 404, because `website.mydomain.com` is not (yet) known in our infrastructure. Depending on your downtime tolerance (clients receiving 404) for a few minutes, there are 2 cases:
```

Comment:
```suggestion
By doing this, clients that already have the `website.mydomain.com` DNS record cached locally will continue to hit `51.15.x.x`, until the TTL expires. New clients (or those whose cache has expired) will start to hit `example-website.functions.fnc.fr-par.scw.cloud`. **However**, until the custom domain is not configured on the Serverless Container, these requests will end up in 404, because `website.mydomain.com` is not (yet) known in our infrastructure. Depending on your downtime tolerance (clients receiving 404) for a few minutes, there are 2 cases:
```